### DISCUSSION TITLE
Is ARM64 host supported?

### DISCUSSION THREAD

**Discussion by kernel-cyrus** [arighi/virtme-ng#44] [General]
Hello, can I use it on ARM based Macbook?

**Comment by arighi** [comment]
Hm... that is a good question. I've never tried to run virtme-ng on an arm macbook (I don't have one :sweat_smile: ), but theoretically it's supported, as long as you're running linux and qemu/kvm installed. And it would be really cool to say that we properly support the ARM macbook, so I'm more than happy to help if you find issues.

Have you tried to install virtme-ng from source? Or even run a `./vng -r` from the source dir directly? Do you get any error?

**Comment by sidkumar99** [comment]
I am trying to run virtme-ng on aarch64 Fedora 40 VM on a macbook pro M3 host and I am getting the following error trying to run vng-r

`qemu-system-aarch64: gic-version=host requires KVM
`
I belive this could be due to not passing the --enable-kvm flag to qemu which seems to be needed on aarch64 hosts although I am not sure about this.

**Reply by kobataiwan** [reply]
kvm needs msr but it doesn't exist on arm64.
```
$ sudo /usr/sbin/kvm-ok
[sudo] password for parallels: 
INFO: /dev/kvm does not exist
HINT:   sudo modprobe kvm
modprobe: FATAL: Module msr not found in directory /lib/modules/6.8.0-40-generic
```

Tried "vng --disable-kvm", still got the same error
```
$ vng --disable-kvm
qemu-system-aarch64: gic-version=host requires KVM
$ vng --version
virtme-ng 1.25
```
